Bugfix/allow false use tz settings

shmoon-kr opened the pull request:
Replace localtime() function with datetime.now() when USE_TZ=False

gqlauth commented:
<!-- ID: 878ae1db-766f-49c7-a1a8-59f7be1fee8f -->
### Release Notes
<details>
<summary>Just wanted to say</summary>

![success](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExMTlmYjI2N2M0Yjk3YzQwOGZjOTYzYWRlNjQwNjkwNWJiZmI2MzhjMyZlcD12MV9pbnRlcm5hbF9naWZzX2dpZklkJmN0PWc/1Z02vuppxP1Pa/giphy.gif)

</details>
Release type: patch

Add UserStatus migration of existing User data
Replace localtime() function with datetime.now()

nrbnlulu commented on gqlauth/migrations/0002_alter_userstatus_options.py:
```diff
@@ -1,8 +1,17 @@
 from django.db import migrations
 
 
+def set_userstatus(apps, schema_editor):
```
a way to test this would be to add a user and disconnect [this](https://github.com/nrbnlulu/strawberry-django-auth/blob/42b996af8aacfc6e152afeb2bf87e46a99c026ee/gqlauth/user/signals.py#L6) signal. call this function and see if the user has a status.

nrbnlulu replied:
A way to test this would be to disconnect [this](https://github.com/nrbnlulu/strawberry-django-auth/blob/42b996af8aacfc6e152afeb2bf87e46a99c026ee/gqlauth/user/signals.py#L6) receiver, create a user and see if he has a status.

nrbnlulu commented on gqlauth/jwt/types_.py:
```diff
@@ -183,7 +184,7 @@ def authenticate(cls, info: Info, input_: ObtainJSONWebTokenInput) -> "ObtainJSO
                 UserStatus.unarchive(user)
             if status.verified or app_settings.ALLOW_LOGIN_NOT_VERIFIED:
                 # successful login.
-                user.last_login = localtime()
```
If you can add a test for this as well, just run this
https://github.com/nrbnlulu/strawberry-django-auth/blob/42b996af8aacfc6e152afeb2bf87e46a99c026ee/tests/test_login.py#L100
with `USE_TZ=False`

shmoon-kr commented:
I added test code but test code for migration doesn't work. The error message was like following:

FAILED tests/test_login.py::TagsTestCase::test_tags_migrated - django.db.utils.NotSupportedError: SQLite schema editor cannot be used while foreign key constraint checks are enabled. Make sure to disable them before entering a transaction.atomic() context because SQLite does not support disabling them in the middle of a multi-statement transaction.

I don't know how to fix it. So, I dropped migration script. Please review the PR and consider merging.
